persistence/sql/migrations/sql/20210817181232000000_unique_credentials.sqlite3.down.sql

```diff
@@ -1 +1,2 @@
-CREATE UNIQUE INDEX "identity_credential_identifiers_identifier_nid_uq_idx" ON "identity_credential_identifiers" (nid, identifier);
```

Comment:
We usually do not change existing migrations. This is fine because it's only SQLite, but was this on purpose?

Reply:
This may be necessary later for the PK change on identity_credential_identifiers, but it is not relevant in this PR. Removed.